src/CMakeLists.txt

```diff
@@ -1,19 +1,41 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-project(Iaito VERSION 1.0.0)
+project(iaito VERSION 1.0.0)
 
-set(CMAKE_CXX_STANDARD 11)
+set(QT_MIN_VERSION "5.6.0")
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+include(CheckCXXCompilerFlag)
+include(FeatureSummary)
+include(GenerateExportHeader)
+
+find_package(Qt5 ${QT_MIN_VERSION} NO_MODULE REQUIRED
+	WebEngine
+	WebEngineWidgets
+)
+
+find_package(Radare2 1.4.0 REQUIRED)
```

Comment:
yes also code in main file checks for version of radare2 which is overheard.
1. should not compile at first if it doesnt have radare2 version as required.
2. for distribution packaging radare2 version to checked as iaito's runtime dependecy instead of complaining after install.

* This may be different for users who are never going to use distribution packaging and just use it from git. Decision entirely depends on project maintainers.